src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1065,11 +1065,15 @@ def config_diagnostics(resource_group_name, name, level=None,
         application_logs = ApplicationLogsConfig(fs_log)
 
     http_logs = None
-    if web_server_logging is not None:
-        enabled = web_server_logging
-        # 100 mb max log size, retenting last 3 days. Yes we hard code it, portal does too
-        fs_server_log = FileSystemHttpLogsConfig(100, 3, enabled)
-        http_logs = HttpLogsConfig(fs_server_log)
+    server_logging_option = web_server_logging or docker_container_logging
+    if  server_logging_option:
+        # TODO: az blob storage log config currently not in use, will be impelemented later
```

Comment:
in the `params.py`, you should remove the `storage` from the choices list
``` python
register_cli_argument('webapp log config', 'web_server_logging', help='configure Web server logging', **enum_choice_list(['off', 'storage', 'filesystem']))

```

Reply:
And the whole code can be simplified, maybe?
```python
server_logging_option = web_server_logging or docker_container_logging
if  server_logging_option:
    http_logs = FileSystemHttpLogsConfig(100, 3, server_logging_option!='off')
```

Reply:
I would like to keep the previous code here as to make it simpler for when the az blob storage code is implemented.

Reply:
I agree.